node/src/isolate/external.cpp

```diff
@@ -0,0 +1,200 @@
+#include "cocaine/detail/isolate/external.hpp"
+#include "cocaine/idl/isolate.hpp"
+
+#include <cocaine/context.hpp>
+#include <cocaine/dynamic.hpp>
+#include <cocaine/errors.hpp>
+#include <cocaine/logging.hpp>
+#include <cocaine/engine.hpp>
+
+#include <cocaine/rpc/session.hpp>
+
+#include <cocaine/traits/dynamic.hpp>
+
+#include <blackhole/logger.hpp>
+#include <asio/ip/tcp.hpp>
+
+using asio::ip::tcp;
```

Comment:
Do this inside namespace to avoid symbol pollution (especially for our compilers, which happily exports those symbols).